form.html

```diff
@@ -0,0 +1,136 @@
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <img class="logo" src="img/karma-logo.svg" alt="logo" />
+        <img
+          class="menu-burger"
+          src="img/menu-hamburger.svg"
+          alt="menu-burger"
+        />
+        <ul>
+          <li>
+            <a href="#" class="link-strong">Meet Karma</a>
+          </li>
+          <li><a href="#">How it Works</a></li>
+          <li><a href="#">Store</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Help</a></li>
+          <li><a href="#">Login</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <form>
+        <h1 class="form_title">Order your Karma wifi device today!</h1>
+        <div class="first_last_name">
+          <div class="first_last">
+            <label for="first_name">First name *</label>
+            <input id="first_name" type="text" minlength="2" required />
+          </div>
+          <div class="first_last last_name">
+            <label for="last_name">Last name *</label>
+            <input id="last_name" type="text" minlength="2" required />
+          </div>
+        </div>
+        <div class="address">
+          <label for="address">Address *</label>
+          <input id="address" type="text" required />
+
+          <label for="address2">Address 2</label>
+          <input id="address2" type="text" required />
+        </div>
+
+        <div class="city_postcode">
+          <div class="city__postcode">
+            <label name="city" for="city">City *</label>
+            <select class="city" id="city">
+              <option value="" class="select_city">Select your city...</option>
+              <option value="london">London</option>
+              <option value="oxford">Oxford</option>
+              <option value="bath">Bath</option>
+              <option value="birmingham">Birmingham</option>
+              <option value="bristol">Bristol</option>
+              <option value="cambridge">Cambridge</option>
+              <option value="chelmsford">Chelmsford</option>
+              <option value="leeds">Leeds</option>
+            </select>
+          </div>
+          <div class="city__postcode postcode">
+            <label for="postcode">Postcode *</label>
+            <input class="postcode" id="postcode" type="text" required />
+          </div>
+        </div>
+
+        <div class="feature_colour">
+          <h4 class="feature_radio_colour">Select colour</h4>
+          <div class="features">
+            <div class="feature_radio orange">
+              <input id="feature_colour_orange" type="radio" />
+              <label for="feature_colour_orange">Karma Orange</label>
+            </div>
+
+            <div class="feature_radio">
+              <input id="feature_colour_grey" type="radio" />
+              <label for="feature_colour_grey">Space Grey</label>
```

Comment:
* Since `name` attribute is absent, you can now incorrectly select both radio boxes